conan/cli/commands/remote.py

```diff
@@ -116,14 +104,16 @@ def remote_update(conan_api, parser, subparser, *args):
                            help="Insert the remote at a specific position in the remote list")
     subparser.set_defaults(secure=None)
     args = parser.parse_args(*args)
-    if not (args.url is not None or args.secure is not None):
+    if args.url is None and args.secure is None and args.index is None:
```

Comment:
Quick quiestion. You deleted --secure from the add command, but then what's different in the update command to still needed it too?

Reply:
You might need to update a "insecure" remote and make it "secure", and this update must be explicit, it is not good that it will update it to make it secure by default (because you are changing the URL or the index). While the default for ``conan remote add`` is being secure, always, unless you explicitly opt-out, and thus ``--secure`` in ``conan remote add`` is completely unnecessary.

Reply:
That makes lots of sense, thank you!